utils/Theme.ts

```diff
@@ -0,0 +1,21 @@
+import { PaletteColorOptions } from '@mui/material';
+import { TypeBackground } from '@mui/material/styles/createPalette';
+
+export const primary: PaletteColorOptions  = {
+    light: "#ffffff",
+    main: "#ffffff",
+    dark: "#cccccc",
+    contrastText: "#000000"
+}
+
+export const secondary: PaletteColorOptions = {
+    light: "#a975ff",
+    main: "#7447d5",
+    dark: "#3d19a3",
+    contrastText: "#ffffff"
+}
+
+export const background: Partial<TypeBackground> = {
+    default: "#eee9fa",
+    paper: "#000000"
+}
```

Comment:
* The width of the rectangle of the input of the search bar is too large while there is no focus
* The police of "Login" is not as bold, and it is in uppercase

Reply:
What do you mean by "The police of "Login" is not as bold, and it is in uppercase"?

Reply:
The one in the proto : 
![image](https://user-images.githubusercontent.com/59285665/145582634-f45eb2d0-120c-4c50-8b3d-5b4d9669e2b0.png)
The one in the code :
![image](https://user-images.githubusercontent.com/59285665/145582691-08faaa35-4463-4d43-936d-b01cb6119070.png)

Reply:
As you can see, yours is in uppercase, and the police is lighter

Reply:
<img width="157" alt="image" src="https://user-images.githubusercontent.com/15985204/145583583-9fd03480-aa1a-437a-871c-9aa0986f310f.png">
Is it better now ?

Reply:
Yes !